### PR TITLE
Feat: new Datazoom quickstart (partner)

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New Datazoom quickstart. 1 Dashbaord, 2 images, install plan.

## Pre merge checklist

- [X ] Did you check you NRQL syntax? - Does it work?
- [ X] Did you check your dashboard image quality? -  Do they look good?
- [ X] Did you include an InstallPlan and Documentation reference?
- [ X] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ X] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/datazoom/config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Name of the quickstart (required)
+name: datazoom-quickstart
+
+# Displayed in the UI (required)
+title: Datazoom Quickstart
```
Comment (jpvajda) [inline quickstarts/datazoom/config.yml:5]:
```suggestion
title: Datazoom 
```

Comment (jpvajda) [reply quickstarts/datazoom/config.yml:5]: Thanks for the contribution!  I don't think we need to call it a quickstart in the title as it's implicitly a quickstart.

File: quickstarts/datazoom/config.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+# Long-form description of the quickstart (required)
+description: |
+  The Datazoom quickstart provides a fast and easy launching point into video metrics built on top of New Relic One Dashboards, allowing you to gain insights about your viewers, content, ads and your platform's performance. Customize the dashboard to use only metrics that fit your goals. Edit metric calculations and tailor them to your needs.
```
Comment (jpvajda) [inline quickstarts/datazoom/config.yml:9]:
```suggestion
  The Datazoom quickstart provides a fast and easy launching point into video metrics built on top of New Relic One dashboards, allowing you to gain insights about your viewers, content, ads and your platform's performance. Customize the dashboard to use only the metrics that fit your goals and easily edit metric calculations and tailor them to your needs.
```

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Comment by jcountsNR** [comment]
@jpvajda , good callout. I updated them, let me know how it looks.

**Comment by jcountsNR** [comment]
@jpvajda  any updates on this one? It should be good to go.

**Review by jpvajda — APPROVED** [review]

File: quickstarts/datazoom/config.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Name of the quickstart (required)
+name: datazoom-quickstart
```
Comment (jpvajda) [inline quickstarts/datazoom/config.yml:2]:
```suggestion
name: datazoom
```

Comment (jpvajda) [reply quickstarts/datazoom/config.yml:2]: We are recommended we don't use quickstart in our names and titles as it's redundant.

File: quickstarts/datazoom/config.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  - featured
+  - newrelic partner
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
+  - third-party-datazoom
+
+documentation:
+  - name: Installation Docs
```
Comment (jpvajda) [inline quickstarts/datazoom/config.yml:40]:
```suggestion
  - name: Installation docs
```

Comment (jpvajda) [reply quickstarts/datazoom/config.yml:40]: gotta watch that sentence case.  😉

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Comment by jpvajda** [comment]
@jcountsNR I caught a few more things, I made some suggestions.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**